code/modules/swarmers/swarmer_objs.dm

```diff
@@ -106,8 +108,24 @@
 		- YOU AND YOUR DRONES HAVE A STUN EFFECT ON MELEE.  YOU ARE ALSO ARMED WITH A DISABLER PROJECTILE, USE THESE TO PREVENT ORGANICS FROM HALTING YOUR PROGRESS\n\
 		GLORY TO !*# $*#^</span>")
 	swarmer.forceMove(get_turf(src))
+	LAZYADD(swarmerlist, swarmer)
+	RegisterSignal(swarmer, COMSIG_PARENT_QDELETING, .proc/remove_swarmer, swarmer)
 	processing_swarmer = FALSE
 
+/**
+ * Removes a swarmer from the beacon's list.
+ *
+ * Removes the swarmer from our list.
+ * Called specifically when a swarmer is about to be destroyed, so we don't have any null references.
+ * Arguments:
+ * * mob/swarmer - The swarmer to be removed from the list.
+ */
```

Comment:
force needs documenting

Reply:
According to Willox on Discord, because force is just a param of a signal handler and that's there it comes from, it doesn't really need documentation since we don't do it elsewhere.  Its not actually used anywhere in here.

Reply:
It's still a weird situation to have one argument documented and the other not - but duplicating documentation from wherever the force param on the `COMSIG_PARENT_QDELETING` signal lives doesn't seem right either (especially if it isn't even used.)

Reply:
All args need to be documented, even if its just going to say "parameter of signal handler" since that will at least prompt you to look at a signal vs. just wondering wtf it's doing.